src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static int NUM_OF_TASK = 0;
+    private static String LINE = "____________________________________________________________";
+    private static final String BYE_COMMAND = "bye";
+    private static final String LIST_COMMAND = "list";
+    private static final String DONE_COMMAND = "done";
+    private static final String TODO_COMMAND = "todo";
+    private static final String EVENT_COMMAND = "event";
+    private static final String DEADLINE_COMMAND = "deadline";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        runChatbot();
+    }
+
+    private static void runChatbot() {
+        Task[] Tasks = new Task[100];
+
+        Scanner in = new Scanner(System.in);
+        String arr[] = getCommand(in);
+
+        while (true) {
+            runCommand(arr, Tasks, in);
+            arr = getCommand(in);
+        }
+    }
+
+    private static String[] getCommand(Scanner in) {
+        String s = in.nextLine();
+        String arr[] = s.split(" ", 2);
+        return arr;
+    }
+
+    private static void runCommand(String[] arr, Task[] Tasks, Scanner in)  {
+        System.out.println(LINE);
+        switch (arr[0]) {
+        case (BYE_COMMAND):
+            printExitMessage();
+        case (LIST_COMMAND):
+            printList(Tasks);
+            break;
+        case (DONE_COMMAND):
+            try {
+                int taskNum = Integer.parseInt(arr[1]);
+                taskNum--;
+                Tasks[taskNum].setDone(true);
+                printDone(Tasks[taskNum]);
+            } catch (ArrayIndexOutOfBoundsException e) { //If arr[1] does not exist
```

Comment:
Looking at the Java Code Standards, it's better to make the comments in the next single line.